doc/integrator/multi_organization.rst.tmpl

```diff
@@ -95,6 +102,43 @@ The following roles should be created and completed as needed
 - org2-registered
 - org2-intranet
 
+Internationalization
+--------------------
+
+We should have one localizations file for each instance, for that we should:
+
+Add this rule in the `geoportal/Makefile`:
+
+.. code::
+
+   .PHONY: update-client-po
+   update-client-po:
+       cd .. && SUFFIX=-org1 INTERFACES=desktop-org1,mobile-org1 \
+           update-po $(USER_ID) $(GROUP_ID) $(LANGUAGES)
+       cd .. && SUFFIX=-org2 INTERFACES=desktop-org2,mobile-org2 \
+           update-po $(USER_ID) $(GROUP_ID) $(LANGUAGES)
+
+This will correctly filter the themes and the layers but not the layer groups.
+
+All available environment variable:
+
+ - ``INTERFACE``: List of interfaces we want to use.
+ - ``THEME_FILTER``: Regular expression used to filter the themes.
+ - ``GROUP_FILTER``: Regular expression used to filter the layer groups.
+ - ``WMSLAYER_FILTER``: Regular expression used to filter the WMS layers.
+ - ``WMTSLAYER_FILTER``: Regular expression used to filter the WMTS layers.
+
+
+Modify the `Dockerfile` like that:
+
+.. code::
+
+   - RUN build-l10n "<package>"
+   + RUN build-l10n --suffix=-org1 --suffix=-org2 "<package>"
+
+
+This will use the ``lang_urls_suffix`` defined above.
```

Comment:
At runtime, I do not really understand what this sentence make here.

Reply:
:-/
=>
For the runtime in the ``interfaces_config`` we will use the ``lang_urls_suffix`` to define witch interface
use witch localization file, see how it's defined above.